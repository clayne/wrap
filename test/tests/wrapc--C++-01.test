wrapc | /dev/null | | hello_01.cpp | 0
