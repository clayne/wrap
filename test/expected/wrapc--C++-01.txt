//
// C++ is a general-purpose programming language.  It has imperative, object-
// oriented and generic programming features, while also providing facilities
// for low-level memory manipulation.
//
// Before the initial standardization in 1998, C++ was developed by Bjarne
// Stroustrup at Bell Labs since 1979, as an extension of the C language as he
// wanted an efficient and flexible language similar to C, which also provided
// high-level features for program organization.
//
#include <iostream>

int main() {
  std::cout << "hello, world\n";
}
